src/options.ts

```diff
@@ -15,6 +15,14 @@ export interface RollupPluginSbomOptions {
      * uses the internal file emitter to write files.
      */
     outDir?: string;
+    /**
+     * The base filename for the SBOM files, defaults to 'bom'
+     */
+    outFilename?: string;
+    /**
+     * The formats to output, defaults to ['json', 'xml']
+     */
+    outFormats?: string[];
```

Comment:
I'd add a concrete union type for the valid values instead of a string array to prevent any errors within TS itself.

Reply:
sure.